uploader/s3.go

```diff
@@ -87,6 +90,11 @@ func (u *S3Uploader) Upload(ctx context.Context, local string, remote string, ct
 		}
 	}()
 
+	// Record number of bytes being uploaded.  At this point, file must exist.
+	if stat, err := os.Stat(local); err == nil {
+		u.metrics.Counter("titus.executor.S3Uploader.uploadMB", int(stat.Size()/1024/1024), nil)
```

Comment:
I think this needs tags or the only information it will tell me is that an agent uploaded some megabytes to S3 during some time frame.

Reply:
what tags can you think of?  Filename may not be a good choice if it rotates and have a numerical or timestamp suffix.  I assume the standard tags already includes container ID, etc.

Reply:
I was just thinking container ID. Does the `u.metrics.Counter` method automatically add container information?

Reply:
nf.node is usually set to the container's task ID from my understanding. I might suggest just asking the insights team all the the nf.* tags they associate with task metrics and use those.